conda/api.py

```diff
@@ -6,8 +6,7 @@
 from .common.constants import NULL
 from .core.package_cache_data import PackageCacheData as _PackageCacheData
 from .core.prefix_data import PrefixData as _PrefixData
-from .core.solve import (DepsModifier as _DepsModifier, Solver as _Solver,
-                         UpdateModifier as _UpdateModifier)
+from .core.solve import (DepsModifier as _DepsModifier, Solver as _Solver, UpdateModifier as _UpdateModifier)
```

Comment:
Your change will break flake8, unfortunately.  An alternate solution would be to break it up across 2 lines:

```suggestion
from .core.solve import DepsModifier as _DepsModifier, Solver as _Solver
from .core.solve import UpdateModifier as _UpdateModifier
```